docs/api/commands/events.md

```diff
@@ -6,4 +6,4 @@ This command is currently unimplemented. Follow [DOCKER-78](http://smartos.org/b
 
 ## Related
 
-- Insert a list of related Docker and CloudAPI methods here
\ No newline at end of file
+- [`MachineAudit`](https://apidocs.joyent.com/cloudapi/#MachineAudit) and `GET /my/machines/:id/audit` in CloudAPI
```

Comment:
I was under the impression that there was no CLI for this one when looking at the CloudAPI docs. I'll fix this.